srcpkgs/crud/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'crud'
+pkgname=crud
+version=0.1
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="pkg-config"
+makedepends="libX11-devel libXext-devel"
+short_desc="Prints the geometry of a selected region to stdout"
+maintainer="Rubén Santos <kojicomics@cocaine.ninja>"
+license="MIT"
+homepage="https://github.com/fosu/crud"
+distfiles="https://github.com/fosu/crud/archive/${version}.tar.gz"
+checksum=0d4061e8ec7f3abde891ceeb982b3303436a8e74047bd04b4501ed43d4797848
+
+do_install() {
+	vbin crud
```

Comment:
gnu-makefile runs make install which should install the program

Reply:
The build failed with the [original make install](https://github.com/fosu/crud/blob/e979fca0050398f303ae3375232075e1cd46742c/Makefile#L14):
`install -m755 crud $(DESTDIR)$(PREFIX)/bin/crud`

This is the _xbps-src_ output:
```
=> crud-0.1_1: running do_install ...
install -m755 crud /destdir//crud-0.1/usr/bin/crud
install: cannot create regular file '/destdir//crud-0.1/usr/bin/crud': No such file or directory
make: *** [Makefile:14: install] Error 1
=> ERROR: crud-0.1_1: do_install: '${make_cmd} STRIP=true PREFIX=/usr DESTDIR=${DESTDIR} ${make_install_args} ${make_install_target}' exited with 2
=> ERROR:   in do_install() at common/build-style/gnu-makefile.sh:40
```
I wasn't able to fix that, so I wrote the _do_install_

Reply:
Simple, it is trying to install to a non-existent directory.

just do 

```
pre_build() {
    sed -i 's|install|install -d|' Makefile
}
```

Reply:
Thanks, it's working now.